src/ext/property-panel/data.ts

```diff
@@ -26,6 +28,69 @@ function isTotalsVisible(itemData: EngineAPI.IHyperCubeDimensionDef, _: unknown,
   return idx === 0 || idx >= noOfLeftDims;
 }
 
+// TODO: scope this out to common repo, don't see any differences in properties of default values atm
+const columnResize = {
+  type: {
+    type: "string",
+    component: "dropdown",
+    ref: "qDef.columnWidth.type",
+    translation: "Object.Table.Column.Width",
+    options: [
+      {
+        value: ColumnWidthType.Auto,
+        translation: "Common.Auto",
+      },
+      {
+        value: ColumnWidthType.FitToContent,
+        translation: "Object.Table.Column.FitToContent",
+      },
+      {
+        value: ColumnWidthType.Pixels,
```

Comment:
Actually that comment about pixel type should be here and I guess I kind of make a bit more sense when switching type. This does not set the actual pixel value on the properties.